src/scripts/main.js

```diff
@@ -1,3 +1,246 @@
 'use strict';
 
-// write code here
+const headerTitle = document.querySelectorAll('th');
+const sortedRows = document.querySelectorAll('tbody tr');
+const sortedRowsToArr = Array.from(sortedRows);
+const docBody = document.querySelector('body');
+const tBody = document.querySelector('tbody');
+
+let lastSortedColumnIndex = null;
+let isAsc = true;
+
+const newForm = document.createElement('form');
+
+newForm.classList.add('new-employee-form');
+
+docBody.appendChild(newForm);
+
+const labelName = document.createElement('label');
```

Comment:
It would be nice to move form creation logic to a separate function
Also, you can create variable with array of form inputs config, _example_:

```
const inputsConfig = [
 { type: 'select', name: 'office', ... }
]
```
and map it to avoid logic duplication

Reply:
Also, try to make form look like on example:

![image](https://github.com/user-attachments/assets/6838ac01-91d9-49db-ab9d-17a7290f8991)